fairseq/models/speech_to_text/berard.py

```diff
@@ -132,7 +132,7 @@ def build_encoder(cls, args, task):
             lstm_size=args.lstm_size,
             dropout=args.dropout,
         )
-        if getattr(args, "load_pretrained_encoder_from", None):
+        if getattr(args, "load_pretrained_encoder_from", None) is not None:
```

Comment:
nit: is not None condition check is not needed I guess?
```suggestion
        if getattr(args, "load_pretrained_encoder_from", None):
```